api/agent/agent.go

```diff
@@ -1161,9 +1137,7 @@ func (a *agent) runHotReq(ctx context.Context, call *call, state ContainerState,
 		break
 	}
 
-	if isEvictable {
-		a.evictor.UnregisterEvictor(evictor)
-	}
+	a.evictor.UnregisterEvictor(evictor)
```

Comment:
this will get called in defer when the function returns, is it ok to call it twice?

Reply:
yes, it's got checks/validations internally.